build/assets/scripts/low-latency-hooks.sh

```diff
@@ -1,35 +1,38 @@
 #!/usr/bin/env bash
 
+JQ="/usr/bin/jq"
+IP="/usr/sbin/ip"
+
 mask="${1}"
 [ -n "${mask}" ] || { logger "${0}: The rps-mask parameter is missing" ; exit 0; }
 
-pid=$(jq '.pid' /dev/stdin 2>&1)
+pid=$(${JQ} '.pid' /dev/stdin 2>&1)
 [[ $? -eq 0 && -n "${pid}" ]] || { logger "${0}: Failed to extract the pid: ${pid}"; exit 0; }
 
-ns=$(ip netns identify "${pid}" 2>&1)
+ns=$(${IP} netns identify "${pid}" 2>&1)
```

Comment:
Let's use double quotes here for both consistency and safeness: https://tldp.org/LDP/abs/html/quotingvar.html